src/index.js

```diff
@@ -1,12 +1,14 @@
 import 'babel-polyfill';
 import 'source-map-support/register';
 
-import debug from 'debug';
+import debugLog from 'debug';
 import fs from 'graceful-fs';
 import mkdirp from 'mkdirp';
```

Comment:
got it, so it sounds like maybe we should add debug to the packages.json file unless there is a reason not to do that